tests/test_backend.py

```diff
@@ -77,6 +77,7 @@
     random_uniform = tf.compat.v1.random_uniform
     fused_batch_norm = tf.compat.v1.nn.fused_batch_norm
     dropout = tf.compat.v1.nn.dropout
+    quantize_and_dequantize = tf.compat.v1.quantization.quantize_and_dequantize
```

Comment:
just wondering - why only tf-1.15? We should be able to make this work for other versions as well, right?

Reply:
Added support for TF2 as well now. The reason this doesn't work TF <=1.14 does not have narrow range argument to restrict the quantization range to [-127, 127]. TF <=1.14 has a default quantization range of [128, 128]. ONNX Quantize seems to be using [-127, 127] by default. Due to this slight change in the range and no argument in ONNX to control the range, the test case will fail for the extreme elements. For the testcase in the code, there are 3 mismatched elements for TF <=1.14. This is due to the range mismatch. 

>  Mismatched elements: 3 / 18 (16.7%)
Max absolute difference: 0.04724407
Max relative difference: 0.00787401
 x: array([[[-6.047244, -6.047244],
        [-6.047244, -6.      ],
        [-5.007874, -4.015748]],...
 y: array([[[-6.      , -6.      ],
        [-6.      , -6.      ],
        [-5.007874, -4.015748]],...

Reply:
Added support for TF2 as well now. The reason this doesn't work TF <=1.14 does not have `narrow_range` argument to restrict the quantization range to [-127, 127]. TF <=1.14 has a default quantization range of [-128, 127]. ONNX Quantize seems to be using [-127, 127] by default. Due to this slight change in the range and no argument in ONNX to control the range, the test case will fail for the extreme elements. For the testcase in the code, there are 3 mismatched elements for TF <=1.14. This is due to the range mismatch. 

>  Mismatched elements: 3 / 18 (16.7%)
Max absolute difference: 0.04724407
Max relative difference: 0.00787401
 x: array([[[-6.047244, -6.047244],
        [-6.047244, -6.      ],
        [-5.007874, -4.015748]],...
 y: array([[[-6.      , -6.      ],
        [-6.      , -6.      ],
        [-5.007874, -4.015748]],...

Reply:
got it. Maybe we can come up with some solution for this later.